CMakeLists.txt

```diff
@@ -45,7 +45,9 @@ FetchContent_Declare(
 # Enable legacy mode for CPP_BINDGEN for users of GT 1.0
 # TODO remove in GT 2.0
 set(CPP_BINDGEN_GT_LEGACY ON)
+set(BUILD_TESTING OFF)
 FetchContent_MakeAvailable(cpp_bindgen)
+unset(BUILD_TESTING)
```

Comment:
This looks dangerous to me, because it assumes that `BUILD_TESTING` is a cache variable (at least I think that's the only case where this might still execute the tests below). I would restore the `BUILD_TESTING` variable, especially because the variable might be set in the parent directory, so we want to respect that for the rest of the file.

Reply:
Felix did a similar workaround here (which I think should be better): https://github.com/GridTools/gridtools/pull/1361/files#diff-af3b638bc2a3e6c650974192a53c7291R48

Reply:
Is there a way to check if the variable exists in the local scope? Because Felix' case will set one in the local scope which might not have been there before.